deepchecks/tabular/checks/data_integrity/is_single_value.py

```diff
@@ -31,17 +31,21 @@ class IsSingleValue(SingleDatasetCheck):
     ignore_columns : Union[Hashable, List[Hashable]] , default: None
         Columns to ignore, if none given checks based
         on columns variable.
+    ignore_nan : bool, default True
+        whether to ignore NaN values in a column or not
     """
 
     def __init__(
         self,
         columns: Union[Hashable, List[Hashable], None] = None,
         ignore_columns: Union[Hashable, List[Hashable], None] = None,
+        ignore_nan: bool = True,
```

Comment:
why the default is True? until now it was False

Reply:
> By default: ignore Nan

Itay asked for this